src/us/kbase/narrativejobservice/sdkjobs/SDKLocalMethodRunner.java

```diff
@@ -668,8 +710,17 @@ public void run() {
                 } catch (Exception ignore) {
                     System.err.println("Error shutting down callback server: " + ignore.getMessage());
                 }
+            try{
+                final URI dockerURI = getURI(config,
+                        NarrativeJobServiceServer.CFG_PROP_AWE_CLIENT_DOCKER_URI,
+                        true);
+                new DockerRunner(dockerURI).killSubJobs();
```

Comment:
Is this the section that replaces the canceljob() calls in the new helper functions?

Reply:
Yes, here and in finishJobPrematurely